manual/src/cmds/intf-c.etex

```diff
@@ -702,12 +702,20 @@ type) corresponds to the "Some" constructor.
 \subsection{ss:c-int-ops}{Operations on integers}
 
 \begin{itemize}
-\item "Val_long("\var{l}")" returns the value encoding the "long int" \var{l}.
-\item "Long_val("\var{v}")" returns the "long int" encoded in value \var{v}.
-\item "Val_int("\var{i}")" returns the value encoding the "int" \var{i}.
-\item "Int_val("\var{v}")" returns the "int" encoded in value \var{v}.
-\item "Val_bool("\var{x}")" returns the OCaml boolean representing the
-truth value of the C integer \var{x}.
+\item "Val_long("\var{n}")" returns the value encoding the C integer \var{n} of
+type "intnat". Despite its name, note that "intnat" is not necessarily the same
+size as a C "long" (e.g., on Windows, "long" is always 32-bit wide, while
+"intnat" has the size of a pointer). This operation can be applied to other C
+integer types, with the argument casted to "intnat" before conversion.
+\item "Long_val("\var{v}")" returns the C integer of type "intnat" encoded in a
+value \var{v}.
+\item "Val_int("\var{i}")" returns the value encoding of the C integer \var{i}
+of type "int".  It coincides with "Val_long("\var{i}")".
```

Comment:
I'm not fond of the "it coincides with ..." comments, which add weight to the documentation (sometimes repeating the implementation) and I'm not sure there are matching benefits.